packages/astro/src/cli/check/index.ts

```diff
@@ -85,7 +85,10 @@ export async function check(
 			commandName: 'astro check',
 			usage: '[...flags]',
 			tables: {
-				Flags: [['--help (-h)', 'See all available flags.']],
+				Flags: [
+					['--watch', 'Check input files in watch mode.'],
```

Comment:
```suggestion
					['--watch', 'Watch Astro files for changes and re-run checks.'],
```

I wonder if this would better explain the behaviour `--watch` enables 🤔

Reply:
You're right. I loosely followed `tsc --watch`'s description but I think this change better explains the `astro check --watch` behavior.